tests/test_selector_csstranslator.py

```diff
@@ -2,6 +2,11 @@
 Selector tests for cssselect backend
 """
 import unittest
+
+import cssselect
+import pytest
+from pkg_resources import parse_version
```

Comment:
Deprecated, adding the `packaging` third-party package and using its `Version` class instead should do the trick.